src/components/mobile/molecules/ReportModal/ReportModal.tsx

```
@@ -0,0 +1,74 @@
+              type="button"
+              value={option.value}
+              onClick={() => {
+                setReportReason(option.value);
+                setOtherReason('');
+              }}
+              css={[
+                S.buttonStyling,
+                option.value === reportReason && S.selectedButtonStyling,
+              ]}
+            >
+              {option.label}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)